flang/test/Lower/OpenMP/debug_info_conflict.f90

```diff
@@ -0,0 +1,16 @@
+! Tests that there no debug-info conflicts arise because of DI attached to nested
+! OMP regions arguments.
+
+! RUN: %flang -c -fopenmp -g -mmlir --openmp-enable-delayed-privatization=true \
+! RUN:   %s -o - 2>&1 | FileCheck %s
```

Comment:
Nit: I think using `%flang_fc1 -emit-llvm` will also reproduce this problem and it will be more in line with the other tests in this folder instead of using `%flang`. You can use `-debug-info-kind=standalone` for `-g`.